inc/azure_c_shared_utility/uuid.h

```diff
@@ -16,26 +16,26 @@ extern "C" {
 
 #include "azure_c_shared_utility/umock_c_prod.h"
 
-typedef unsigned char UUID[16];
+typedef unsigned char UUID_T[16];
```

Comment:
>UUID_T [](start = 22, length = 6)

I don't particularly like the _T because it's used so often, but I can't come up with something better, so carry on.

Reply:
To be more similar to other *_t types, "uuid_t" seems like a better fit I think.

But it's pure style choice, your changes are good.